src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -299,6 +299,10 @@ internal IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundSwitchExpressionArmOperation((BoundSwitchExpressionArm)boundNode);
                 case BoundKind.ObjectOrCollectionValuePlaceholder:
                     return CreateCollectionValuePlaceholderOperation((BoundObjectOrCollectionValuePlaceholder)boundNode);
+                case BoundKind.FunctionPointerInvocation:
```

Comment:
>case BoundKind.FunctionPointerInvocation: [](start = 16, length = 41)

Please make sure to run all tests with IOperation test hook enabled and confirm that there are no failures related to the "Function pointers" feature.

Reply:
I've run them locally and gone through the failures. Nothing related to this feature.

---
In reply to: [428783559](https://github.com/dotnet/roslyn/pull/44436#discussion_r428783559) [](ancestors = 428783559)